test/RoslynAnalyzer.Tests/RoslynAnalyzer.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <LangVersion>Preview</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MonoBuild)' == ''">
+    <TargetFramework>net5.0</TargetFramework>
+    <DefineConstants>$(DefineConstants);ILLINK</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(MonoBuild)' != ''">
+    <TargetFramework>net472</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit" Version="1.0.1-beta1.*" />
+     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.7.0" />
```

Comment:
The version should come from Arcade

Reply:
This is a bit tricky. We probably want two versions here: Arcade for the tests, hardcoded for the project. This is because analyzers are plugins, so upgrading the required analyzer version effectively reduces your platform support to only newer .NET SDKs/Visual Studio installs.